issue-wanted.cabal

```diff
@@ -13,14 +13,17 @@ copyright:           2018 Kowainik
 category:            Web, Application
 build-type:          Simple
 extra-doc-files:     README.md
-                   , CHANGELOG.md
-tested-with:         GHC == 8.4.3
+                     CHANGELOG.md
+tested-with:         GHC == 8.6.5
 
-library
-  hs-source-dirs:      src
-  exposed-modules:     Prelude
-                       IW
-                       IW.Search
+source-repository head
+  type:                git
+  location:            https://github.com/kowainik/issue-wanted.git
+
+common common-options
+  build-depends:       base-noprelude ^>= 4.12.0.0
+                     , relude ^>= 0.5.0
+                     , github >= 0.21
```

Comment:
At the moment not all stanzas using `github` dependency, so it's better to add only dependencies that are used everywhere.
And also, the upper boundary is not specified. It is much better to have all the boundaries specified for the libraries.

By the way, `github-0.21` is very broken, so I would suggest using an older version. We are using `github-0.20` in another project.

Reply:
Okay got it. I will remove `github` from the common stanza. Should the boundary be like 
`github >= 0.20 && < 0.21` or just `github == 0.20`?

Reply:
I used github == 0.20

Reply:
Good! Until the new non-broken version is out we can use this one. We will keep an eye on this 🙂